pitch.md

```diff
@@ -46,24 +92,34 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There aren't any other DSLs in this domain that I know of. Automated coffee machines already have a GUI that lets you customize your drink in this way but
+
+1. It almost certainly doesn't compile to a DSL (I guess **maybe** it does, but I highly doubt it).
+2. The "language" has limited expressiveness. You can choose a base drink, and then modify the amount of coffee, sugar, and milk to add, but that's about it. The language I'm proposing would allow you to create a drink from the bottom up. 
 
 ## The Project
 This section examines whether the idea makes for a good CS 111 project.
 
-
 ### Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+I think I'll be spending a lot of time making language design decisions, but also a fair amount of time implementing the language. The free free-form nature of the language (in particular, the fact that you could probably make programs of arbitrary length) means that parsing will be something of a challenge - I'll probably have to get really familiar with parser combinators. Luckily, once I have the intermediate representation of a program, execution should be pretty easy - just make the relevant library calls with the parameters extracted from the program. 
```

Comment:
I feel like the implementation is just iterating through an instruction list, pre-computing the names of each recipe would make the iteration incredibly simple. I don't really see why the implementation would take that long.